docs/status.md

```diff
@@ -0,0 +1,201 @@
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/crash/main.yml?label=Crash CLI" loading="lazy"></a>
+<a href="https://github.com/crate/croud/actions/workflows/main.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/croud/main.yml?label=Croud CLI" loading="lazy"></a>
+</td>
+</tr>
+
+</tbody>
+</table>
+
+
+## Drivers
+
+CI outcomes for a range of client drivers connecting your applications to CrateDB.
+
+### CrateDB
+Drivers and dialects maintained by Crate.io.
+
+<a href="https://github.com/crate/crate-python/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/crate-python/tests.yml?label=crate-python" loading="lazy"></a>
+<a href="https://github.com/crate/crate-pdo/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/crate-pdo/tests.yml?label=crate-pdo" loading="lazy"></a>
+<a href="https://github.com/crate/crate-dbal/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/crate-dbal/tests.yml?label=crate-dbal" loading="lazy"></a>
+<a href="https://github.com/crate/crate_ruby/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/crate_ruby/tests.yml?label=crate_ruby" loading="lazy"></a>
+<a href="https://github.com/crate/activerecord-crate-adapter/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/activerecord-crate-adapter/tests.yml?label=activerecord-crate-adapter" loading="lazy"></a>
+
+### PostgreSQL
+Standard PostgreSQL drivers for different languages.
+
+<a href="https://github.com/pgjdbc/pgjdbc/actions/workflows/main.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/pgjdbc/pgjdbc/main.yml?label=pgjdbc" loading="lazy"></a>
+<a href="https://github.com/npgsql/npgsql/actions/workflows/build.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/npgsql/npgsql/build.yml?label=npgsql" loading="lazy"></a>
+<a href="https://github.com/psycopg/psycopg2/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/psycopg/psycopg2/tests.yml?label=psycopg2" loading="lazy"></a>
+<a href="https://github.com/psycopg/psycopg/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/psycopg/psycopg/tests.yml?label=psycopg3" loading="lazy"></a>
+<a href="https://github.com/MagicStack/asyncpg/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/MagicStack/asyncpg/tests.yml?label=asyncpg" loading="lazy"></a>
+<a href="https://github.com/brianc/node-postgres/actions/workflows/ci.yml">
+    <img src="https://img.shields.io/github/actions/workflow/status/brianc/node-postgres/ci.yml?label=node-postgres" loading="lazy"></a>
+
+
```

Comment:
```suggestion
```{note}
Please note that those are drivers maintained by community authors and other
maintainers, and reflect the build status of the development head. It doesn't
mean integration with GA releases isn't stable. This is reflected within the
topmost section of this page.
\```

```

Reply:
Done: Added to the bottom of the page, slightly rephrased.